src/widgets/helper/ChannelView.cpp

```diff
@@ -2043,6 +2043,41 @@ void ChannelView::handleLinkClick(QMouseEvent *event, const Link &link,
                                        SettingsDialogPreference::Accounts);
         }
         break;
+        case Link::JumpToChannel: {
+            // Get all currently open pages
+            QSet<SplitContainer *> openPages;
+
+            auto &nb = getApp()->windows->getMainWindow().getNotebook();
+            for (int i = 0; i < nb.getPageCount(); ++i)
+            {
+                openPages.insert(
+                    static_cast<SplitContainer *>(nb.getPageAt(i)));
+            }
+
+            for (auto *sc : openPages)
+            {
+                auto splits = sc->getSplits();
+
+                // Find channel with mention in all opened channels
```

Comment:
Update comment phrasing a bit, maybe something along the lines of:
`Search for channel matching link in page/split container`

Reply:
Fixed in 012f98be